src/main/resources/anniv5Resources/localization/eng/Cardstrings.json

```diff
@@ -65,275 +65,5 @@
   "${ModID}:FromMyHat": {
     "NAME": "From My Hat",
     "DESCRIPTION": "Gain !B! Block. NL Add a random Attack into your hand. NL It gains Retain."
-  },
```

Comment:
The remaining entries in this file are for the starter cards and cards for a demo pack that appears to have been commented out. Probably good to remove those entries at some point, but I didn't want to take too much of a detour from the main point of this PR.